packages/autorest.python/autorest/codegen/models/imports.py

```diff
@@ -312,71 +329,3 @@ def add_msrest_import(
                         ImportType.LOCAL,
                         typing_section,
                     )
-
-    @property
-    def import_core(self) -> str:
-        return "azure.core" if not self.code_model.options["unbranded"] else "corehttp"
-
-    @property
-    def import_core_exceptions(self) -> str:
-        return f"{self.import_core}.exceptions"
```

Comment:
`corehttp` is not stable which means its module structure may change in the future. So I think it is better to handle all the import difference between unbranded and azure. Once change comes, we only need to change `imports.py` instead of change in multi files.

Reply:
In favor of https://github.com/Azure/autorest.python/pull/2232

Reply:
the issue here though is you're having to create dummy file imports in order to access the name of the import in order to pass to another file import. This kind of circular logic isn't great, and 90% of the imports are the same for corehttp and azure-core, they just have a different prefix, and that's how it's expected to stay